mcs/tools/corcompare/mono-api-info.cs

```diff
@@ -1390,10 +1395,13 @@ public void DoOutput (XmlWriter writer, IList<ICustomAttributeProvider> provider
 			writer.WriteEndElement (); // attributes
 		}
 
-		static Dictionary<string, object> CreateAttributeMapping (CustomAttribute attribute)
+		Dictionary<string, object> CreateAttributeMapping (CustomAttribute attribute)
 		{
 			Dictionary<string, object> mapping = null;
 
+			if (!state.TypeHelper.TryResolve (attribute))
+				return mapping;
```

Comment:
Try an resolve the attribute before processing it. This may be defined in this assembly, but the properties may be using types that will not be found.